sktalk/corpus/parsing/cha.py

```diff
@@ -44,3 +79,8 @@ def _clean_utterance(cls, utterance):
         utterance = re.sub(r" {2}", " ", utterance)
         utterance = re.sub(r"[ \t]{1,5}$", "", utterance)
         return utterance
+
+    @staticmethod
+    def _clean_timing(timing):
+        timing = timing.split("_")
+        return [int(t) for t in timing] if len(timing) == 2 else None
```

Comment:
I suppose you could further simplify this part by having two separate match groups in the `TIMING_REGEX` above, e.g.:

```python
    TIMING_REGEX = `r"\D(?P<start>\d{1,9})_(?P<end>\d{1,9})\D"`
```
This would make the `split` and length check redundant, and you could just return a tuple (or list) like this:

```python
    return int(match.group("start")), int(match.group("end"))
```

However, that would again require changes in other places, so perhaps leave it as it is now.